### DISCUSSION TITLE
Is there a command line that can turn off the screen but operate it?

### DISCUSSION THREAD

**Discussion by wanababy** [NetrisTV/ws-scrcpy#249] [Q&A]
Scrcpy has the command "scrcpy - sw", but you have to open a window, which cannot be operated in pure command line mode.
Can Linux have command line for screen operation? thank you!

**Comment by drauggres** [comment]
Hi. It is hard to understand what do you want.
Ws-scrcpy should have similar feature to `-S/--turn-screen-off` flag of the original scrcpy. There is a button and a checkbox under `More` menu (three dots).
See https://github.com/NetrisTV/ws-scrcpy/pull/69

**Comment by wanababy** [comment]
I mean that I want to control the switch of the screen through a terminal or a sh file, so that I can control multiple mobile phones at one time or automate the process. I don't want to click on the page one by one.

**Comment by zexry619** [comment]
is there any solution for this? I want to run ws-scrcpy with the screen off
